.travis.yml

```diff
@@ -1,35 +1,34 @@
-# This is a common .travis.yml for generating library release zip files for
-# CircuitPython library releases using circuitpython-build-tools.
-# See https://github.com/adafruit/circuitpython-build-tools for detailed setup
-# instructions.
-
 dist: trusty
 sudo: false
 language: python
 python:
-    - "3.6"
-
+- '3.6'
 cache:
-    pip: true
-
+  pip: true
 deploy:
   provider: releases
```

Comment:
There should be a - here before provider. I think thats why the release files are missing.

Reply:
There's no `- here` in the instructions or in any of the `.travis.yml` files I can find, including libs I didn't PR.

Please confirm this is necessary. 

If so, we need to update the instructions and I will go back through all the libs we've done and add it.

Reply:
Oh you mean a `-` before provider. I see. It apparently needs to be added each time, it's not automatic. I'll go back through everything and make sure it's there in the ones done before I started as well.